faiss-search/scann/scann.py

```diff
@@ -81,7 +83,9 @@ def build_advanced_index(self, vecs: 'np.ndarray'):
             tree(self.num_leaves, self.num_leaves_to_search, self.training_sample_size).\
```

Comment:
sorry I did not express it correctly, you do retrun `self._index`, but you need to assign it, here `_index` is just a local variable